UnhollowerBaseLib/Attributes/Il2CppImplementsAttribute.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+
+namespace UnhollowerBaseLib.Attributes
+{
+    [AttributeUsage(AttributeTargets.Class, AllowMultiple = false)]
+    public class Il2CppImplementsAttribute : Attribute
+    {
+        public readonly Type[] Interfaces;
```

Comment:
```suggestion
        public Type[] Interfaces { get; }
```

Reply:
This doesn't match the style of the other attributes.

Reply:
It doesn't matter, unhollower's style isn't really good. Generally you should never make public fields in C# because you can't just redirect them later without breaking stuff when you want to rename a field for example.